gammapy/makers/utils.py

```diff
@@ -248,18 +294,31 @@ def make_map_background_irf(
     else:
         image_geom = geom.to_image()
         d_omega = image_geom.solid_angle()
-
-    coords = _get_fov_coords(
-        pointing=pointing,
-        irf=bkg,
-        geom=geom,
-        use_region_center=use_region_center,
-        obstime=obstime,
-    )
-    coords["energy"] = broadcast_axis_values_to_geom(geom, "energy", False)
-
-    bkg_de = bkg.integrate_log_log(**coords, axis_name="energy")
-    data = (bkg_de * d_omega * ontime).to_value("")
+    if bkg.has_offset_axis or bkg.fov_alignment == FoVAlignment.RADEC:
+        data = evaluate_bkg(
+            pointing, ontime, bkg, geom, use_region_center, obstime, d_omega
+        )
+    if bkg.fov_alignment == FoVAlignment.ALTAZ:
+        endtime = obstime + ontime
+        data = np.zeros(geom.data_shape)
+        while obstime < endtime:
+            # Evaluate the step time needed to have FoV rotation of less than fov_rotation_error_limit
+            # Minimum step of 1 second to avoid infinite computation very close to zenith
+            steptime = max(
+                time_limit_rotation(
+                    fov_rotation_error_limit, pointing.get_altaz(obstime)
+                ),
+                1 * u.s,
+            )
+            steptime = (
+                steptime
+                if steptime < endtime - obstime - 1 * u.s
+                else endtime - obstime
+            )
```

Comment:
```suggestion
            duration = np.min(steptime, obstime)
```

Reply:
`obstime` is the time corresponding to the start of the interval. It is not a duration.
The full obs duration is `ontime`.

A correct replacement would be : 
steptime = np.min(steptime, endtime - obstime)
if we ignore the minimum duration of the step that I added as 1 * u.s